app/src/main/java/com/github/se/eduverse/ui/timetable/DetailsTasksScreen.kt

```diff
@@ -68,18 +64,7 @@ fun DetailsTasksScreen(
   }
 
   Scaffold(
-      topBar = {
-        TopNavigationBar(task.name, navigationActions) {
-          IconButton(
-              onClick = {
-                timeTableViewModel.deleteScheduled(task)
-                navigationActions.goBack()
-              },
-              modifier = Modifier.testTag("deleteButton")) {
-                Icon(Icons.Default.Delete, contentDescription = "Delete")
-              }
-        }
-      },
+      topBar = { TopNavigationBar(navigationActions, screenTitle = null) },
```

Comment:
Same as above :
Here you shouldn't have remove the delete button. You can copy-paste this after TopNavigationBar(...) :
```
{
          IconButton(
              onClick = {
                timeTableViewModel.deleteScheduled(task)
                navigationActions.goBack()
              },
              modifier = Modifier.testTag("deleteButton")) {
                Icon(Icons.Default.Delete, contentDescription = "Delete")
              }
        }
```